package/gluon-config-mode-contact-info/i18n/de.po

```diff
@@ -16,12 +16,16 @@ msgstr "Kontakt"
 msgid ""
 "Please provide your contact information here to allow others to contact "
 "you. Note that this information will be visible <em>publicly</em> on "
-"the internet together with your node's coordinates."
+"the internet together with your node's coordinates. This information is "
+"not required to operate a node. If you chose to enter data, it will be "
+"stored on this node and can be deleted by yourself at any time."
 msgstr ""
 "Bitte hinterlege hier einen Hinweis, um anderen zu ermöglichen, "
 "Kontakt mit dir aufzunehmen. Beachte, dass dieser Hinweis auch "
 "<em>öffentlich</em> im Internet, zusammen mit den Koordinaten "
-"deines Knotens, einsehbar sein wird."
+"deines Knotens, einsehbar sein wird. Für den Betrieb sind diese "
+"Informationen nicht erforderlich. Eine Speicherung erfolgt auf diesem Router. "
+"Diese Daten können Durch Dich in diesem Menü eigenständig gelöscht werden."
```

Comment:
* "Durch" klein
* "Dich" klein (im Rest der Texte schreiben wir "Du" auch klein)
* "in diesem Menü" -> "auf dieser Seite" oder so

Reply:
done.

Reply:
sorry for this, somehow the correction didn't make it into the commit. This is fixed now for good.